src/main/java/net/minecraftforge/registries/ForgeRegistry.java

```diff
@@ -871,6 +883,45 @@ public Snapshot makeSnapshot()
         return ret;
     }
 
+    private class RegistryCodec implements Codec<V>
+    {
+        @Override
+        public <T> DataResult<Pair<V, T>> decode(DynamicOps<T> ops, T input)
+        {
+            if(ops.compressMaps())
+            {
+                return ops.getNumberValue(input).flatMap(n ->
+                {
+                    int id = n.intValue();
+                    if(ids.get(id) == null)
```

Comment:
Formatting: space before opening parenthesis.
```suggestion
                    if (ids.get(id) == null)
```